cmd/start-amazon-cloudwatch-agent/path.go

```diff
@@ -80,22 +78,8 @@ func startAgent(writer io.WriteCloser) error {
 func generateMergedJsonConfigMap() (map[string]interface{}, error) {
 	ctx := context.CurrentContext()
 	setCTXOS(ctx)
-	ctx.SetInputJsonFilePath(jsonConfigPath)
-	ctx.SetInputJsonDirPath(jsonDirPath)
+	ctx.SetInputJsonFilePath(paths.JsonConfigPath)
+	ctx.SetInputJsonDirPath(paths.JsonDirPath)
 	ctx.SetMultiConfig("remove")
 	return cmdutil.GenerateMergedJsonConfigMap(ctx)
 }
-
-func init() {
-	jsonConfigPath = paths.AgentDir + "/etc/" + JSON
-	jsonDirPath = paths.AgentDir + "/etc/" + paths.JsonDir
-	envConfigPath = paths.AgentDir + "/etc/" + ENV
-	tomlConfigPath = paths.AgentDir + "/etc/" + TOML
-	commonConfigPath = paths.AgentDir + "/etc/" + COMMON_CONFIG
-	yamlConfigPath = paths.AgentDir + "/etc/" + YAML
-
-	agentLogFilePath = paths.AgentDir + "/logs/" + AGENT_LOG_FILE
-
-	translatorBinaryPath = paths.AgentDir + "/bin/" + paths.TranslatorBinaryName
-	agentBinaryPath = paths.AgentDir + "/bin/" + paths.AgentBinaryName
-}
```

Comment:
init() functions have been moved into tool/paths/path_unix.go and tool/paths/path_windows.go